From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Fri, 10 Jun 2022 14:21:13 +0200
Subject: Invalidate components public key

This prevents usage of any signed component

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 .../component_updater/pki_metadata_component_installer.cc      | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/component_updater/pki_metadata_component_installer.cc b/chrome/browser/component_updater/pki_metadata_component_installer.cc
--- a/chrome/browser/component_updater/pki_metadata_component_installer.cc
+++ b/chrome/browser/component_updater/pki_metadata_component_installer.cc
@@ -72,7 +72,8 @@ const char kGoogleOperatorName[] = "Google";
 // The extension id is: efniojlnjndmcbiieegkicadnoecjjef
 const uint8_t kPKIMetadataPublicKeySHA256[32] = {
     0x45, 0xd8, 0xe9, 0xbd, 0x9d, 0x3c, 0x21, 0x88, 0x44, 0x6a, 0x82,
-    0x03, 0xde, 0x42, 0x99, 0x45, 0x66, 0x25, 0xfe, 0xb3, 0xd1, 0xf8,
+    // this fingerprint is changed on purpose in Bromite
+    0x01, 0x02, 0x03, 0x04, 0x05, 0x66, 0x25, 0xfe, 0xb3, 0xd1, 0xf8,
     0x11, 0x65, 0xb4, 0x6f, 0xd3, 0x1b, 0x21, 0x89, 0xbe, 0x9c};
 
 const base::FilePath::CharType kCTConfigProtoFileName[] =
--
2.25.1
